logging/logging.go

```diff
@@ -687,6 +687,12 @@ type HTTPRequest struct {
 	// validated with the origin server before being served from cache. This
 	// field is only meaningful if CacheHit is true.
 	CacheValidatedWithOriginServer bool
+
+	// CacheFillBytes is the number of HTTP response bytes inserted into cache. Set only when a cache fill was attempted.
+	CacheFillBytes int64
+
+	// CacheLookup tells whether or not a cache lookup was attempted.
+	CacheLookup bool
```

Comment:
I'm still confused how this gets populated. Does the user need to fill this in? Or is it from a header?

Reply:
This is being populated in line 721.

Reply:
I checked this and I don't think we should rely on the headers of http request object of golang.
The fields in question are `CacheLookup` and `CacheFillBytes`. For `CacheFillBytes` there is no equivalent header with the same semantic that I could find in http protocol spec. For `CacheLookup`, there are 2 headers that cover this: `X-Cache` and `X-Cache-Lookup`, however both don't have well defined semantic that is **guaranteed** to mean the same when used in different proxies and CDNs. 
More details here: https://lyte.id.au/2014/08/28/x-cache-and-x-cache-lookupheaders/

I propose that we keep the current change intact - user will be responsible to set the fields via HTTPRequest of Entry object whenever they want to use it in log entry payload. Just like they already do with, for example, CacheHit field of the HTTPRequest.

Reply:
Sounds good. Thank you for looking into it!